doc/developers-guide/OpenSSL-and-FIPS.md

```diff
@@ -34,40 +32,40 @@ export CFLAGS=-I/home/vagrant/openssl/include
 
 ### How to enable/disable FIPS mode
 
-In the release directory there is file `etc/app.config`. FIPS mode is an option of crypto application. In order to enable/disable FIPS mode add following section to `app.config`
+Find `etc/app.config` in the release directory. 
+FIPS mode is an option of the crypto application. 
+In order to enable/disable it, add the following section to `app.config`:
 
 ```erlang
 {crypto, [{fips_mode, Value}]},
 ```
 
-Where `Value` is `true` or `false`.
+where `Value` is either `true` or `false`.
 
-### How to check if FIPS mode is enabled
+### How to check if the FIPS mode is enabled
 
 #### Log message
 
-When MongooseIM starts it prints following log message if FIPS mode is enabled
+When MongooseIM starts, it prints the following log message if FIPS mode is enabled
 
 ```
 2015-02-25 14:30:54.501 [warning] <0.242.0>@mongoose_fips:do_notify:37 FIPS mode enabled
 ```
 
 #### Run-time check
 
-In MongooseIM's console run function:
+Run the following function in the MongooseIM console:
 
 ```erlang
 mongoose_fips:status().
 ```
 
 ### Cipher suites difference
 
-A test using `cipher_suites_test.sh` script (to be found in tools
-directory) was run against MongooseIM with FIPS mode enabled and
-disabled. The OpenSSL version was: `OpenSSL 1.0.1j-fips`.
+A test using a `cipher_suites_test.sh` script (available in the tools directory) can be performed on MongooseIM with FIPS mode enabled and disabled. 
+We've used `OpenSSL 1.0.1j-fips`.
 
-Below there is a list of all possible cipher suites when **FIPS** mode was
-**enabled**:
+Here are all the cipher suites returned when the **FIPS** mode is **enabled**:
```

Comment:
I know it's an output of `cipher_suites_test.sh` but I'd rather use word "available", not "returned".